test/test_imagej.py

```diff
@@ -82,6 +82,25 @@ def testImageMath(self):
             result.append(itr.next().get())
         self.assertEqual(result, correct_result)
 
+    def testPluginsLoadUsingPairwiseStitching(self):
+        macro = """
+        
+        newImage("Tile1", "8-bit random", 512, 512, 1);
+        newImage("Tile2", "8-bit random", 512, 512, 1);
+        run("Pairwise stitching", "first_image=Tile1 second_image=Tile2 fusion_method=[Linear Blending]""" + \
+                """fused_image=Tile1<->Tile2 check_peaks=5 display_fusion x=0.0000 y=0.0000""" +\
+                """ registration_channel_image_1=[Average all channels]""" +\
+                """ registration_channel_image_2=[Average all channels]");"""
```

Comment:
The above command can be simplified to:

```
run("Pairwise stitching", "first_image=Tile1 second_image=Tile2");
```

All omitted options are the default choices and can safely be omitted.

Reply:
Thanks for the tip.  I've chopped off the extra text and simplified the command to your line.